### PR TITLE
Feat: FlexJobs Scrapper added

### PR DESCRIPTION
**Pull request by AKASH722**
# Description

> The changes made in this pull request include the creation of a FlexJobs scrapper, implementation of unit tests for the scrapper, and updates to the documentation.

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

> image

> image

> image

> image

## Additional Notes/Comments

> I have created the flexjobs scrapper please check that out. 
> I want Indeed website scrapper next, so should i open another issue, or work under the same as both are job scrappers

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/scrape_up/flexjobs/flexjobs.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+class FlexJobs:
+    """
+    A class to scrape job listings from FlexJobs website based on search query and location.
```
Comment (nikhil25803) [inline src/scrape_up/flexjobs/flexjobs.py:7]: The docstring of the class and the methods are not properly formatted. You can check modules like `GitHub` and `IMDB` for a reference to how these modules have their docstring formatted.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/flexjobs/flexjobs.py (hunk 1 above)
Comment (AKASH722) [reply src/scrape_up/flexjobs/flexjobs.py:7]: Okay i will fix that out.

Comment (AKASH722) [reply src/scrape_up/flexjobs/flexjobs.py]: @nikhil25803 changed the docstrings could you check now

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Great work @AKASH722  !!
Do not forget to
+ Share your contribution on socials and tag Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Pull request merged**